GOVERNANCE.rst

```diff
@@ -335,16 +346,51 @@ sktime uses a “consensus seeking” process for making decisions. The
+-  For a proposed changes to be approved via lazy consensus, one of the following
+   two conditions must be satisfied:
+    - approval by at least one core developer and no rejection by a core developer (veto right).
+      The approval required for this condition must be by a core developer different from a proposer.
+    - approval in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  For a proposed change to be rejected via lazy consensus, one of the following two conditions
+   must be satisfied:
+    - rejection by at least one core developer, and no acceptance by a core developer.
+    - rejection in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  Formal statements on proposed changes must be located with the proposal on GitHub,
+   usually in the same pull request that contains the proposal, or the related issue.
+   Formal statements in this sense are approvals/rejections, and statements
+   about whether the proposal should be considered a "major change", as below.
 -  Approvals and rejections can be expressed as +1 and -1 comments,
-   respectively.
--  Core developers are expected to give reasonable time to others to
-   give their opinion on the pull request if they’re not confident
-   others would agree.
--  More important changes that impact the full project require a more
+   respectively, or any written comments
+   containing "I formally approve/reject" in reference to the proposal.
+   If the proposed change has the form of a PR, approval must be in the form of a GitHub PR approval.
+-  Major changes to the project require a more
    detailed analysis and a consensus that is both explicit and informed.
-   These changes require an :ref:`steps`.
+   A major change always require a worked out :ref:`steps`.
+   A change is major in the above sense
+   if at least three core developers explicitly state that the proposal is a major change
+   and requires especial consideration, in the location.
+-  Proposers are expected to give reasonable time for consideration, that is,
+   time and opportunity for core developers to review and
+   give their opinion on the contribution.
+   Ten working days excluding week-ends constitute "reasonable time" in the above sense.
+   For pull requests, the period resets at every new change made to the PR.
+   It starts only when all GitHub checks pass.
+-  If the "reasonable time" period elapses and no approval or rejection has been expressed on a PR,
+   the PR is scheduled for decision, at the top of the next regular core developer meeting's agenda
+   in which less than seven PR are already on the agenda. If no regular core developer meeting
+   is scheduled within the two calendar weeks after the period elapses,
+   the core developers must schedule one within those two weeks.
```

Comment:
I'd rephrase this as in my comment, since this only applies to PRs by core devs 
```suggestion
-  If a PR by a core developer has not received any approvals or rejections within ten working days excluding weekends, the core developer who opened the PR has the right to put the PR up for discussion at the top agenda in the next regular core developer meeting and receive a review from at least one of the other core developers. 
```

Reply:
this would make the process less clear:

* how do we prevent too long core dev meetings since there are so many PR?
* how do we prevent that no core dev meeting is planned, is it then indefinitely stalled?
* what does "review" mean? If it is a GitHub review, why is a core dev meeting needed? As opposed to a reviewer being appointed with the duty to review?

Reply:
clarified that the dev meeting appoints a reviewer.